### PR TITLE
Feature/1.0.0

### PR DESCRIPTION
**Pull request by ghbm-itk**

### CODE REVIEW

File: robot_framework/linear_framework.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def main():
             error_count += 1
             handle_error(f"Process Error #{error_count}", error, None, orchestrator_connection)
 
-    reset.clean_up(orchestrator_connection)
-    reset.close_all(orchestrator_connection)
+    # reset.clean_up(orchestrator_connection)
+    # reset.close_all(orchestrator_connection)
```
Comment (rimi-itk) [inline robot_framework/linear_framework.py]: Remove these statements if they're not needed.

File: robot_framework/reset.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+from robot_framework import config
 
 
 def reset(orchestrator_connection: OrchestratorConnection) -> None:
     """Clean up, close/kill all programs and start them again. """
     orchestrator_connection.log_trace("Resetting.")
-    clean_up(orchestrator_connection)
-    close_all(orchestrator_connection)
+    # clean_up(orchestrator_connection)
+    # close_all(orchestrator_connection)
```
Comment (rimi-itk) [inline robot_framework/reset.py]: Remove dead code.

File: robot_framework/sap/fpo4.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    session.findById("wnd[1]/tbar[0]/btn[8]").press()
+
+    # Set outputstyring
+    session.findById("wnd[0]/usr/tabsTABSTRIP_TAB_B1/tabpTB12").select()
+    session.findById("wnd[0]/usr/tabsTABSTRIP_TAB_B1/tabpTB12/ssub%_SUBSCREEN_TAB_B1:RFKKOP04:1002/ctxtP_NM_OPL").text = "STEPHAN"
+
+    # Search
+    session.findById("wnd[0]/tbar[1]/btn[8]").press()
+
+    file_path = os.path.join(os.getcwd(), "fpo4.txt")
```
Comment (rimi-itk) [inline robot_framework/sap/fpo4.py:41]: Out of curiosity: Where do the files `fpo4.txt` and `fplka.txt` come from?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: robot_framework/sap/fpo4.py (hunk 3 above)
Comment (ghbm-itk) [reply robot_framework/sap/fpo4.py:41]: The robot creates the files as part of the process. "fpo4" and "fplka" are names of SAP transactions. Don't ask me what they mean...
I just decided to call the files the same as the transaction that creates them. They are created and deleted almost instantly, but that had to have some name

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**